runtime/magic-numbers.h

```diff
@@ -1,17 +1,13 @@
-#if 0 /* this header can be included into both Fortran and C */
-Copyright (c) 2019, NVIDIA CORPORATION.  All rights reserved.
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
+#if 0 /*===-- runtime/magic-numbers.h -------------------------------------===*/
```

Comment:
Why not move the comment to a separate line?  I recall some compilers complain about comments after pre-processor directives.

Reply:
I'm only aware of compilers complaining about comments that aren't terminated on the same preprocessor line. I wanted the first line to have the header information, which is the custom in LLVM.

Reply:
Just FYI, the LLVM OpenMP header files don't follow that convention.
f18-llvm-project/openmp/runtime/src/include